java/dataflow-connector-examples/src/main/java/com/google/cloud/bigtable/dataflow/example/HelloWorldWrite.java

```diff
@@ -71,44 +74,51 @@ public void processElement(DoFn<String, Mutation>.ProcessContext c) throws Excep
       c.output(new Put(c.element().getBytes()).addColumn(FAMILY, QUALIFIER, VALUE));
     }
   };
+  // [END bigtable_dataflow_connector_process_element]
 
   /**
    * <p>Creates a dataflow pipeline that creates the following chain:</p>
    * <ol>
-   *   <li> Puts an array of "Hello", "World" into the Pipeline
-   *   <li> Creates Puts from each of the words in the array
-   *   <li> Performs a Bigtable Put on the items in the
+   * <li> Puts an array of "Hello", "World" into the Pipeline
```

Comment:
If you're going to modify these lines, it would be good to fix:

* there should be no space between `<li>` and the text
* `<li>` should be closed with `</li>`

Reply:
Done